src/pages/Main/Best/Best.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import ProductPreview from '../../ProductList/ProductPreview/ProductPreview';
+import '../Best/Best.scss';
+
+class Best extends Component {
+  render() {
+    return (
+      <section className="best">
+        <div className="bestTitle">
+          <img alt="dish" src="/images/Main/dish.png" />
+          <div className="topTitle">
+            <p>MECOOK</p>
+            <p id="best">BEST</p>
+          </div>
+          <div className="botTitle">
+            <p>미쿡에서</p>
+            <p>잘 나가는 메뉴를 한눈에!</p>
+          </div>
+        </div>
+        <div className="bestList">
+          <p id="viewBest">
+            VIEW BEST&nbsp;<i class="fas fa-chevron-right"></i>
+          </p>
+          <ul className="listItems">
+            <li className="food">
+              <ProductPreview />
+            </li>
+            <li className="food">
+              <ProductPreview />
+            </li>
+            <li className="food">
+              <ProductPreview />
+            </li>
+            <li className="food">
+              <ProductPreview />
+            </li>
+            <li className="food">
+              <ProductPreview />
+            </li>
+            <li className="food">
+              <ProductPreview />
+            </li>
```

Comment:
반복되는 부분 map 메서드 활용해서 구현해주세요!

Reply:
map 메서드를 활용해서 새로 수정하였습니다~!
```suggestion
            {menuList &&
              menuList.map(m => {
                const {
                  id,
                  image,
                  type,
                  name,
                  cookingTime,
                  availPeople,
                  like,
                } = m;
                return (
                  <ProductPreview
                    key={id}
                    id={id}
                    image={image}
                    type={type}
                    name={name}
                    cookingTime={cookingTime}
                    availPeople={availPeople}
                    like={like}
                  />
                );
              })}
```